TENETS.md

```diff
@@ -0,0 +1,54 @@
+# MRTK3 Tenets
+
+The following are the guiding principles by which the MRTK3 project abides. This is a living document and the list may be modified over time.
+
+1. [Break large changes into manageable chunks](#break-large-changes-into-manageable-chunks)
+1. [Semantic versioning](#semantic-versioning)
+1. [Support all application stages](#support-all-application-stages)
+1. [Centralized project authority](#centralized-project-authority)
+1. [Get community feedback on large decisions](#get-community-feedback-on-large-decisions)
+1. [Avoid backchannel code reviews](#avoid-backchannel-code-reviews)
+1. [Community contributions are the responsibility of the project](#community-contributions-are-the-responsibility-of-the-project)
+1. [Set clear project guidelines](#set-clear-project-guidelines)
+1. [Be prompt with feedback and anticipated shipping updates](#be-prompt-with-feedback-and-anticipated-shipping-updates)
+1. [Be kind and professional](#be-kind-and-professional)
+
+## Break large changes into manageable chunks
+Every contributor is expected to limit the size of their pull requests to something that can be considered reasonable to review. Excessively large changes are difficult to review and often hide bugs and less than ideal architectures.
+
+## Semantic versioning
+MRTK3 packages are individually versioned and follow semantic versioning rules. Please see [Semantic Versioning 2.0.0](https://semver.org) for details.
+
+The version number for packages must convey meaningful information to customers. When a package's major version number is updated, it can be expected that there are changes that may break existing projects. A minor version update implies addition of non-breaking features.  
+
+## Support all application stages
+MRTK3 is designed to be used by developers of new and existing Unity projects. Features should not be added that limit the abilities of developers to create their applications, regardless of the current stage of development.
+
+## Centralized project authority
+MRTK3 is governed by a steering committee. This committee is comprised of members from multiple companies with a vested interest in the success of MRTK. Decisions are made per the documented [governance](GOVERNANCE.md).
+
+Being an open source project does not mean that everyone has the ability to approve or reject changes. The [maintainers group](MAINTAINERS.md) has the responsibility for determining if a change is appropriate and meets the project guidelines and standards. This is not to imply that the MRTK3 project does not wish to receive code review feedback from non-maintainers, we openly encourage feedback and will take it under advisement when making approval decisions. 
+
+## Get community feedback on large decisions
+The community are our partners and they will have the opportunity to voice their opinions on large decisions. This feedback will be taken into account by the steering committee and weighed against the "needs of the many".
```

Comment:
Would be nice to perhaps indicate which channels are monitored or used by the maintainers to engage with the community on large decisions, so people can ensure they are plugged into discussions.

Reply:
added link to discussions section